actionview/test/ujs/public/test/attach-bindings.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('rails:attachBindings', function() {
+  window.Rails.linkClickSelector += ', a[data-custom-remote-link]';
+  // Hijacks link click before ujs binds any handlers
+  // This is only used for ctrl-clicking test on remote links
```

Comment:
Would it make sense to elaborate on this a little more?

Reply:
This comment was taken from https://github.com/rails/rails/blob/fad500b8c62b053bcd1f8faf2a13f28d3def573f/actionview/test/ujs/views/layouts/application.html.erb#L13-L14. 

Without this is place - the tests hang,  I think as a result of the link being opened in a new tab/window, but I have not been able to verify exactly what is going on yet.